pkg/cmd/util/clientcmd/factory_test.go

```diff
@@ -40,6 +33,7 @@ func TestRunGenerators(t *testing.T) {
 	}
 }
 
+/*
```

Comment:
I don't think this is needed anymore, we switched to using upstream `clientCache`.